BitmarkRegistry/Custom Views/DesignedSegmentControl.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  DesignedSegmentControl.swift
+//  BitmarkRegistry
+//
+//  Created by Macintosh on 7/16/19.
+//  Copyright © 2019 thuyentruong. All rights reserved.
+//  Reference: https://medium.com/@pleelaprasad/creating-a-custom-segmented-control-in-swift-10fede366da1
+//
+
+import UIKit
+
+class DesignedSegmentControl: UIControl {
```

Comment:
`DesignedSegmentControl` this name is quite generic and I cannot guess what's it doing.
Suggest to either have comments or rename this class to be meaningful.

Reply:
ok, thanks.
I can not think another meaningful name cause it's right that the class is quite generic.
so I think I will add some comment like:
DesignedSegmentControl: create our own Segment Control to have ability to implement UI/functionality as design
- supports in Properties tab.
:)